src/utils/deprecate.js

```diff
@@ -25,7 +25,7 @@ const deprecate = function(message, test) {
 
 deprecate._console = typeof console !== 'undefined' ? console : {};
 deprecate._warn = function() {
-  const warn = deprecate._console.warn || deprecate._console.log || function() {};
+  const warn = deprecate._console.warn || deprecate._console.log || Function.prototype;
```

Comment:
Is `Function.prototype` es6 specific? Cause es6 new "methods" wont be translated by babel.